vuepress/docs/v7.0/docs/reference/databases.md

```diff
@@ -1,80 +1,70 @@
+---
+sidebarDepth: 2
+---
 # Manage Entando Databases
 
-## Overview
+Entando currently supports PostgreSQL, MySQL and Oracle database systems. With
+PostgreSQL and MySQL, Entando automatically creates a
+Kubernetes deployment hosting the DBMS. 
+For Oracle and others, Entando
+supports connectivity to [External Databases](../../tutorials/devops/external-db.md). 
 
-Entando’s Docker images currently support three different relational
-database management systems: PostgreSQL, MySQL and Oracle. With
-PostgreSQL and MySQL. By default, Entando’s will automatically create a
-Kubernetes Deployment hosting the database management systems. However,
-for Oracle, and also for other scenarios that may require it, Entando
-supports connectivity to existing external databases. This document
-provides the user with the necessary information to decide how to manage
-Entando’s databases.
+This document provides information about how Entando manages database connectivity.
 
-## Lightweight, low-config Databases with PostgreSQL and MySQL
+## PostgreSQL and MySQL
 
+#### Lightweight & Low-Config 
```

Comment:
just looking online "low-config" doesn't seem to be a thing. here it's meant to be used as a recognizable term. not a huge deal but it would be better to adopt the common terminology of "low config"